core/data/nlu.fr.md

```diff
@@ -1,15 +1,19 @@
-## intent:greet
-- hey
-- hello
-- salut
-- allo
-- bonjour
-- bon matin
-- bonne soirée
-- bon après-midi
-- yo
+## intent: cancel
+- j'ai changé d'idée
+- je ne veux pas m'inscrire
+- annuler
+- je veux pas donner mon numéro
```

Comment:
Should this be a more specific intent like `cancel_enrollment`? We might have other kinds of cancellation (but we could use the same intent with different kind of sentences).

Reply:
Gne... Giving it a specific name while including generic examples would be incorrect with both possible future approaches (many cancels vs one big cancel), so I went with a generic name to avoid having two intents, which is not yet necessary. It points to the second approach but we could change it if it ends up being the wrong one.

Reply:
Since we don't have a context specific NLU, I think it's safer to stick with one cancel intent.